include/QuartzCore/CARenderer.h

```diff
@@ -15,12 +15,12 @@
 //******************************************************************************
 #pragma cone
 
+#import <CoreVideo/CVTime.h>
```

Comment:
Hi Raj,

I originally tried to forward declare the struct but couldn't get it to work given how the original CVTimeStamp is declared.

typedef struct {
    uint32_t version;
    int32_t videoTimeScale;
    int64_t videoTime;
    uint64_t hostTime;
    double rateScalar;
    int64_t videoRefreshPeriod;
    CVSMPTETime smpteTime;
    uint64_t flags;
    uint64_t reserved;
} CVTimeStamp;

Do you know how to forward declare it?

Thanks,

Brent